lust/test/widget_tests/utils_tests/timeHandler_test.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:lust/widgets/utils/timeHandler.dart';
+import 'package:matcher/matcher.dart';
+
+void main() {
+  test("currentEvenHour", () {
+//    expect(TimeHandler.makeHourEven(0), 0);
+//    expect(TimeHandler.makeHourEven(11), 10);
+//    expect(TimeHandler.makeHourEven(24), 24);
+//    expect(ArgumentError("test"), ArgumentError("test"));
+    expect(() => throw ArgumentError(), throwsA(TypeMatcher<ArgumentError>()));
+    expect(TimeHandler.throwException(), throwsA(TypeMatcher<ArgumentError>()));
```

Comment:
@gsocher @RedTo do you know how to write a proper test for exceptions? The test passes, but the second not. I looks like the test does not handle the exception.

Reply:
You can improve the `TimeHandler` a little bit and also solve this problem with the following:

`expect(() => TimeHandler.makeHourEven(25), throwsA(TypeMatcher<ArgumentError>()));`

As you can see, you will hand in a wrong input and your method will throw an error as defined. The `throwException` is not needed. As a rule of thumb: Do not write code (methods) for tests. Sometimes you will need to, but the most of the time it is not necessary.

Reply:
> The throwException is not needed. As a rule of thumb: Do not write code (methods) for tests. Sometimes you will need to, but the most of the time it is not necessary.

Sure, that was only to test the behavior. ;)

But I don't get why that lambda expression is ok but simply invoke the static function not. Could you give me a hint?